### PR TITLE
Feature/versioninfo

### PR DESCRIPTION
**Pull request by ericthegrey**
Hi @GjjvdBurg 

This small PR is about some left-over changes:

- Removes `RecipientId` class which was no longer in use (incorrectly referenced in the `Quote` class).
- Change the last uses of `warnings` and replace with `logging`.

And more importantly, centralize the tests where the database version is relevant into a dedicated class (`VersionInfo`).

### CODE REVIEW

File: signal2html/core.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def get_color(db, recipient_id):
     return color
 
 
-def get_sms_records(db, thread, addressbook, version=None):
+def get_sms_records(db, thread, addressbook, versioninfo=None):
```
Comment (GjjvdBurg) [inline signal2html/core.py:62]: I think this argument isn't used anymore right?

File: signal2html/core.py [hunk 2]
```diff
@@ -125,7 +126,7 @@ def add_mms_attachments(db, mms, backup_dir, thread_dir):
 
 
 def get_mms_records(
-    db, thread, addressbook, backup_dir, thread_dir, version=None
+    db, thread, addressbook, backup_dir, thread_dir, versioninfo=None
```
Comment (GjjvdBurg) [inline signal2html/core.py:129]: same with the ``versionInfo`` argument here

File: signal2html/core.py [hunk 3]
```diff
@@ -185,12 +174,19 @@ def get_mms_records(
 
 
 def populate_thread(
-    db, thread, addressbook, backup_dir, thread_dir, version=None
+    db, thread, addressbook, backup_dir, thread_dir, versioninfo=None
```
Comment (GjjvdBurg) [inline signal2html/core.py:177]: and here

File: signal2html/core.py [hunk 4]
```diff
@@ -221,7 +217,7 @@ def process_backup(backup_dir, output_dir):
         t = Thread(_id=_id, recipient=recipient)
         thread_dir = os.path.join(output_dir, t.sanename)
         populate_thread(
-            db, t, addressbook, backup_dir, thread_dir, version=db_version
+            db, t, addressbook, backup_dir, thread_dir, versioninfo=versioninfo
```
Comment (GjjvdBurg) [inline signal2html/core.py:220]: which means this ``versionInfo`` can be removed too

**Review by GjjvdBurg — CHANGES_REQUESTED** [review]
Nice addition! Looks good to go except for the minor comments I posted

File: signal2html/core.py (hunk 1 above)
Comment (ericthegrey) [reply signal2html/core.py:62]: True, removing

File: signal2html/core.py (hunk 2 above)
Comment (ericthegrey) [reply signal2html/core.py:129]: Well, yes, but it is already needed in #19 (https://github.com/GjjvdBurg/signal2html/pull/19/files#diff-3ce0fcbfb4a5a5cfc47df0642f6e5a035f922740b74d9d3e438319223f1aebc3R173).

File: signal2html/core.py (hunk 3 above)
Comment (ericthegrey) [reply signal2html/core.py:177]: (Could also stay due to #19)

File: signal2html/core.py (hunk 2 above)
Comment (GjjvdBurg) [reply signal2html/core.py:129]: Ah right, makes sense

File: signal2html/core.py (hunk 1 above)
Comment (GjjvdBurg) [reply signal2html/core.py]: :+1:

**Review by GjjvdBurg — APPROVED** [review]

**Comment by GjjvdBurg** [comment]
Thanks again!

**Pull request merged**